README.md

```diff
@@ -3,10 +3,14 @@
  ### Dependencies and  Requirements 
  * JDK 8
  * [twitter4j](http://twitter4j.org/en/) 
- * Twitter Dev account (Note: to see the default account this lab is connected to, go 
- [here](https://twitter.com/HadarRozenberg))
+ * Twitter Dev [account](https://developer.twitter.com/en/portal/register/welcome)
  
 ### Instructions
+* Setup: In the root folder, create a *twitter4j.properties* that includes your credentials. You can make a copy of 
+*twitter4j_example.properties* in the same folder, and remove change the name to twitter4j.properties. 
```

Comment:
instead of moving/ copy to file twitter4j.properties.  Moving it means you then get a git change for deletion which you don't actually want